sections/perl.zsh

```diff
@@ -0,0 +1,39 @@
+#
+# Perl
+#
+# Perl is a general purpose programming language, originally developed for
+# text manipulation.
+# Perl refers to Perl 5.x, where Perl 6+ officially changed the name to Raku.
+#
+# Link: https://www.perl.org/
+# ------------------------------------------------------------------------------
+# Configuration
+# ------------------------------------------------------------------------------
+
+SPACESHIP_PERL_SHOW="${SPACESHIP_PERL_SHOW=true}"
+SPACESHIP_PERL_PREFIX="${SPACESHIP_PERL_PREFIX="$SPACESHIP_PROMPT_DEFAULT_PREFIX"}"
+SPACESHIP_PERL_SUFFIX="${SPACESHIP_PERL_SUFFIX="$SPACESHIP_PROMPT_DEFAULT_SUFFIX"}"
+SPACESHIP_PERL_SYMBOL="${SPACESHIP_PERL_SYMBOL="🐪 "}"
+SPACESHIP_PERL_COLOR="${SPACESHIP_PERL_COLOR="blue"}"
+
+# ------------------------------------------------------------------------------
+# Section
+# ------------------------------------------------------------------------------
+
+# Show current version of Perl
+spaceship_perl() {
+  [[ $SPACESHIP_PERL_SHOW == false ]] && return
+
+  # Show only if perl files or composer.json exist in current directory
+  [[ -n *.pl(#qN^/) || -f composer.json ]] || return
```

Comment:
Use `spaceship::upsearch` for `composer.json` file to find it up in the directory tree.

Reply:
Also, what about:

* `.perl-version`
* `META.(json|yaml)`
* `cpanfile`
* `Makefile.PL`
* `Build.PL`